### DISCUSSION TITLE
Is there any  legecy version available (such as 0.6) for Python 3.7?

### DISCUSSION THREAD

**Discussion by hayatejp** [autogluon/autogluon#3040] [Q&A] [answered]
I have to work the py37 for some project,I know that the AutoGluon 0.7 is not available for that,So is there any method I can install some legecy version such as 0.6 like cuda 11.x?

**Comment by gradientsky** [comment, marked as answer]
[0.6.2](https://pypi.org/project/autogluon/0.6.2/) was the last one supporting python 3.7
you still can do `pip install autogluon==0.6.2`

**Reply by hayatejp** [reply]
OK,Thank you!It works.
